### PR TITLE
Correct WRFv4 workload input paths

### PR DESCRIPTION
**Pull request by sudheerchunduri**
Corrects the workload input paths

### CODE REVIEW

**Comment by rfbgo** [comment]
Hey @sudheerchunduri ! Awesome to see this contribution, thank you

Could you also please update the links in the tests (`test_wrfv4_dry_run`) so they match? (See the CI output for details)

File: var/ramble/repos/builtin/applications/wrfv4/application.py [hunk 1]
```diff
@@ -23,10 +23,10 @@ class Wrfv4(SpackApplication):
                   variants='build_type=dm+sm compile_type=em_real nesting=basic ~chem ~pnetcdf',
                   compiler='gcc9', mpi='impi2018', required=True)
 
-    input_file('CONUS_2p5km', url='https://www2.mmm.ucar.edu/wrf/users/benchmark/v42/v42_bench_conus2.5km.tar.gz',
+    input_file('CONUS_2p5km', url='https://www2.mmm.ucar.edu/wrf/users/benchmark/v422/v42_bench_conus2.5km.tar.gz',
```
Comment (rfbgo) [inline var/ramble/repos/builtin/applications/wrfv4/application.py:26]: Could you also please update the links in the tests (test_wrfv4_dry_run) so they match? (See the CI output for details)

**Review by rfbgo — DISMISSED** [review]

**Review by douglasjacobsen — APPROVED** [review]

**Pull request merged**